### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by ywc689**
v1.8.4 Release: Bugfix for Keepalived.

### CODE REVIEW

File: src/scheduler.c [hunk 1]
```diff
@@ -32,21 +32,18 @@ struct dpvs_role_str {
...
-        { LCORE_ROLE_MASTER,        "lcore_role_master" },
-        { LCORE_ROLE_FWD_WORKER,    "lcore_role_fwd_worker"},
-        { LCORE_ROLE_ISOLRX_WORKER, "lcore_role_isolrx_worker"},
-        { LCORE_ROLE_MAX,           "lcore_role_null"},
+    static const char *role_str_tab[] = {
+        [LCORE_ROLE_IDLE]          = "lcre_role_idle",
+        [LCORE_ROLE_MASTER]        = "lcre_role_master",
+        [LCORE_ROLE_FWD_WORKER]    = "lcre_role_fwd_worker",
+        [LCORE_ROLE_ISOLRX_WORKER] = "lcre_role_isolrx_worker",
+        [LCORE_ROLE_MAX]           = "lcre_role_null"
```
Comment (mscbg) [inline src/scheduler.c:40]: lcore

File: src/ipvs/ip_vs_service.c [hunk 2]
```diff
@@ -1504,7 +1504,7 @@ struct dpvs_sockopts sockopts_svc = {
 
 static int flush_msg_cb(struct dpvs_msg *msg)
 {
-    
+
```
Comment (mscbg) [inline src/ipvs/ip_vs_service.c]: delete empty line

File: tools/keepalived/keepalived/check/ipvswrapper.c [hunk 3]
```diff
@@ -763,6 +763,7 @@ ipvs_set_srule(int cmd, ipvs_service_t *srule, virtual_server_t *vs)
 	srule->user.protocol = vs->service_type;
 	srule->user.bps = vs->bps;
 	srule->user.limit_proportion = vs->limit_proportion;
+        srule->user.conn_timeout = vs->conn_timeout;
```
Comment (mscbg) [inline tools/keepalived/keepalived/check/ipvswrapper.c]: alignment

File: tools/keepalived/keepalived/check/ipvswrapper.c [hunk 4]
```diff
@@ -364,10 +364,12 @@ ipvs_change_needed(int cmd, virtual_server_group_entry_t *vsge, virtual_server_t
 
 		return (count == 0);
 	}
+#if 0
```
Comment (mscbg) [inline tools/keepalived/keepalived/check/ipvswrapper.c:367]: Did nothing for this commit？？

Comment (ywc689) [reply tools/keepalived/keepalived/check/ipvswrapper.c:367]: The codes are controversial. Keep the original codes for references of future debug.

**Pull request merged**